src/app/shared/components/routed-dialog-wrapper/routed-dialog-wrapper.component.ts

```diff
@@ -1,96 +1,74 @@
-/* @format */
 import {
   Component,
   OnInit,
   OnDestroy,
   AfterViewInit,
   ViewChild,
   TemplateRef,
+  Injector,
 } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
-import {
-  DialogRef,
-  Dialog,
-  DialogComponentToken,
-} from '@root/app/dialog/dialog.module';
+import { DialogRef } from '@angular/cdk/dialog';
 import { RouteData } from '@root/app/app.routes';
-import { DialogOptions } from '@root/app/dialog/dialog.service';
-import {
-  HasSubscriptions,
-  unsubscribeAll,
-} from '@shared/utilities/hasSubscriptions';
-import { Subscription } from 'rxjs';
 import { RouteHistoryService } from '@root/app/route-history/route-history.service';
 import { Title } from '@angular/platform-browser';
 import { slideUpAnimation } from '@shared/animations';
+import { Subscription } from 'rxjs';
+import { HasSubscriptions, unsubscribeAll } from '@shared/utilities/hasSubscriptions';
+import { DialogCdkService } from '@root/app/dialog-cdk/dialog-cdk.service';
 
 @Component({
   selector: 'pr-routed-dialog-wrapper',
-  template: ` <ng-template #outletTemplate>
-    <div [@slideUpAnimation]="o.isActivated ? o.activatedRoute : ''">
-      <router-outlet #o="outlet"></router-outlet>
-    </div>
-  </ng-template>`,
+  template: `
+    <ng-template #outletTemplate>
+      <div [@slideUpAnimation]="o.isActivated ? o.activatedRoute : ''">
+        <router-outlet #o="outlet"></router-outlet>
+      </div>
+    </ng-template>
+  `,
   animations: [slideUpAnimation],
 })
 export class RoutedDialogWrapperComponent
   implements OnInit, AfterViewInit, HasSubscriptions, OnDestroy
 {
-  private dialogToken: DialogComponentToken;
-  private dialogOptions: DialogOptions;
-  private dialogRef: DialogRef;
-
-  private closedByNavigate = false;
-
   @ViewChild('outletTemplate') private outletTemplate: TemplateRef<any>;
 
   subscriptions: Subscription[] = [];
+  private dialogRef: DialogRef<any> | null = null;
+  private closedByNavigate = false;
+
   constructor(
     private route: ActivatedRoute,
     private router: Router,
-    private dialog: Dialog,
+    private dialogService: DialogCdkService,
+    private injector: Injector,
     private routeHistory: RouteHistoryService,
-    private title: Title,
+    private title: Title
   ) {}
 
   ngOnInit(): void {}
 
   ngAfterViewInit(): void {
     this.title.setTitle(`${this.route.snapshot.data.title} | Permanent.org`);
 
-    this.dialogToken = (this.route.snapshot.data as RouteData).dialogToken;
+    const dialogToken = (this.route.snapshot.data as RouteData).dialogToken;
+    const component = (this.route.snapshot.data as RouteData).component;
 
-    if (!this.dialogToken) {
+    if (!dialogToken) {
       throw new Error(
-        "RoutedDialogWrapperComponent - missing dialog token on route data, can't open dialog",
+        "RoutedDialogWrapperComponent - missing dialog token on route data, can't open dialog"
       );
     }
```

Comment:
As far as I understand it, the DialogToken type should be completely unused at this point. This means we should be able to remove these lines checking for DialogToken completely, including this throw statement.